vaas/vaas/metrics/handler.py

```diff
@@ -0,0 +1,28 @@
+from datetime import timedelta
+from django.conf import settings
+from vaas.metrics.prometheus import PrometheusMetrics
+from vaas.metrics.statsd import StatsdMetrics
+
+
+class MetricsHandler:
+    def __init__(self) -> None:
+        self.prometheus = PrometheusMetrics()
+        self.statsd = StatsdMetrics()
+
+    def time(self, metric_name: str, value: timedelta) -> None:
+        if settings.STATSD_ENABLE:
+            self.statsd.sum(metric_name, value)
+
+        if settings.PROMETHEUS_ENABLE:
+            self.prometheus.sum(metric_name, value)
+            self.prometheus.client.push()
```

Comment:
If the push is always mandatory maybe it is worth making API for statsd/prometheus consistent (calling push internally in sum/gauge?)

Reply:
I have moved it to PrometheusMetrics. 
Statsd sends metrics in their methods.